data-migrator/core/src/main/java/io/camunda/migration/data/HistoryMigrator.java

```diff
@@ -470,6 +480,18 @@ protected void migrateVariable(HistoricVariableInstance c7Variable) {
     }
   }
 
+  protected void handleVariableInterceptorException(VariableInterceptorException e, String c7VariableId, Date createTime) {
+    HistoryMigratorLogs.skippingHistoricVariableDueToInterceptorError(c7VariableId, e.getMessage());
+    HistoryMigratorLogs.stacktrace(e);
+    markSkipped(c7VariableId, TYPE.HISTORY_VARIABLE, createTime, e.getMessage());
+  }
+
+  protected void handleDecisionInstanceInterceptorException(VariableInterceptorException e, String c7DecisionInstanceId, Date evaluationTime) {
+    HistoryMigratorLogs.skippingDecisionInstanceDueToInterceptorError(c7DecisionInstanceId, e.getMessage());
+    HistoryMigratorLogs.stacktrace(e);
+    markSkipped(c7DecisionInstanceId, TYPE.HISTORY_DECISION_INSTANCE, evaluationTime, e.getMessage());
+  }
+
```

Comment:
🔧 Can be merged. But I have same pattern in [interceptor PR](https://github.com/camunda/camunda-7-to-8-migration-tooling/pull/511/changes#diff-a3868c77d91adc4932fadeba7c21ba02240a33c629add4d0a0f731ccc0a114ecR1150) so I can do it there too.